Gallery/Panels/NCL_panel_4.py

```diff
@@ -30,14 +30,17 @@
 ###############################################################################
 # Read in data:
 
-# Open a netCDF data file using xarray default engine and load the data into
-# xarrays, choosing the 2nd timestamp
-ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc")).isel(time=1)
+# Open a netCDF data file using xarray default engine and save as a variable
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
 
 ###############################################################################
```

Comment:
```suggestion

```

I think you should combine the "Read in data" and "save zonal and meridional..." into one block underneath "Read in data"

Reply:
I agree here